doc/userGuide.md

```diff
@@ -744,15 +744,15 @@ Final configuration:
 -   **Line 10**: The `ListControl` has two _action capabilities_, called "set" and
     "change". There may be different verbs associated with these capabilities and line 10
     shows how to set the list of verbs associated with the "set" capability. This line
-    declares that if the user says "set", "add", or "select", or one of their synonyms,
+    declares that if the user says "set", "select", or one of their synonyms,
     the control should interpret it as an effort to set the value of this control. So, for
     example, if the user says "U: Select fruit please" this control will respond
     `canHandle = true` and set `this.state.value = "fruit"` during `handle()`. Each item
-    in the array is the ID of an slot value for the Action slot type. New action slot
+    in the array is the ID of a slot value for the Action slot type. New action slot
     value IDs should be added for semantically different verbs and synonyms can be added
-    when the is not major semantic difference. So, for example, the action-slot-value for
+    when there are not major semantic differences. So, for example, the action slot value for
     "select" has the synonym "choose". If the user says "U: I choose fruit", it will
-    arrive as `action = "select"` and be recognized by this control. All controls test the
+    arrive as `action = "set"` and be recognized by this control. All controls test the
```

Comment:
nit: `action="select"`

Reply:
Thanks for taking a look, Shreyas. For the nit, this was a change I wasn't totally sure about. The interaction model in the example has the following:
```
  actions: {
      set: [Strings.Action.Set, Strings.Action.Select],
      change: [Strings.Action.Change],
  },
```

And it says that:
> the action slot value for "select" as the synonym "choose." If the user says "U: I choose fruit," it will arrive as `action = "select"`

The cause for my confusion is that I'm reading the interaction model as suggesting that the valid actions are "set" and "change", and that `Strings.Action.Set` maps to the slot value "set" and its synonyms, and that `Strings.Action.Select` maps to "select" and its synonyms. In other words, I read it as meaning that the slot value could be "select" or its synonyms, but the the slot value (regardless of what it is) would get mapped to the `set` action, since it's a member of that key's set. Is that incorrect?

The alternative I can imagine is that `Strings.Action.Select` and `Strings.Action.Set` are both actions, but I'm not sure what you call the `set` and `change` keys at that point.

Thanks again!

Reply:
Shreyas is correct on this point.  NLU will product an intent with action=select and then the Control determines that it should _treat it_ as being the same as action=set.

### Details
So... the idea for our interaction model is that we have an 1:n mapping of control-centric behaviors to user-centric verbs.  The confusion is arising because we use the term 'action' to refer to both.

Using current terminology:  "The control cares about certain actions, e.g. set & change.  These control-actions are associated to action-slot-values and the mapping is 1:many.  Thus the control may associate the set-action to the set-value and the select-value if it wants to.".
NLU produces slot values.  so if the user says 'select' we get an intent with action=select.  IF the user says a synonym of select, e.g. 'choose', nlu still produces an intent with action=select.
Only inside the control code do we look at the mapping and determine that from the Controls point of view we should treat this as being exactly the same as "set".

I think it will make some things easier to comprehend if we change the Control props to call things Capabilities or similar.
```
props:
   capabilityMappings: {
     "set": [Actions.Set, Actions.Select]
  }
```

Was the explanations sufficiently clear?  What do you think of the proposal?

Reply:
Yes, this helps. I think referring to them as capabilities rather than actions makes the distinction clearer.